canonical/charmhub.io
# Construct integration index page

## Done
- construct integrations index page from the folder structure of [interfaces](https://github.com/canonical/charm-relation-interfaces/tree/main/interfaces)
- redirect `/interfaces` and `/interfaces.json` to `integrations` and `/integrations.json` respectively
- update tests
## How to QA
- Go to https://charmhub-io-1821.demos.haus/integrations.json and ensure that interfaces are returned
- Go to https://charmhub-io-1821.demos.haus/integrations and ensure that the interfaces table is displayed as expected, description and category column should be hidden, click on any of the interfaces to go to the interface index page `https://charmhub-io-1821.demos.haus/integrations/<interface-name>`
- Go to https://charmhub-io-1821.demos.haus/interfaces and ensure that it redirects to https://charmhub-io-1821.demos.haus/integrations
- Go to https://charmhub-io-1821.demos.haus/interfaces.json and ensure it redirects to https://charmhub-io-1821.demos.haus/integrations.json
## Issue / Card https://warthogs.atlassian.net/browse/WD-10962
Fixes #

## Screenshots
